src/MediaPresenter.js

```diff
@@ -97,18 +102,19 @@ var ScarletsMediaPresenter = function(options, latency){
 		scope.bufferHeader = null;
 		var bufferHeaderLength = false;
 
-		scope.mediaRecorder = new MediaRecorder(mediaStream, options, scope.polyfill);
+		scope.mediaRecorder = new MediaRecorder(mediaStream, options);
```

Comment:
I think we will need to keep the `scope.polyfill` on there, because when we're using the [recorder polyfill](https://github.com/kbumsik/opus-media-recorder#working-with-a-bundler) someone may want to add custom option for modifying the `workerOptions` 🤔

Reply:
Would it be possible to detect if the native MediaRecorder class or a polyfill? 
On my setup (latest Firefox Dev Edition, Fedora Linux 34) this third param throws error.

Reply:
Hmm, looks like you're right.. it's also throw error when the MediaRecorder was given the third param even it was `undefined` value on Firefox 90, while chromium based browser works normally.

By the way, the polyfill docs said if the `encoderWorker.js` already loaded using the `<script>` tag, we can leave `workerOptions` or `scope.polyfill` empty..

Let's use your implementation 👍

Reply:
what do you about something similar this?

```
  let usingOpusMediaRecorderPolyfill = false;
  if (window.OpusMediaRecorder) {
	  if (options.alwaysUsePolyfill) {
		  usingOpusMediaRecorderPolyfill = true;
	  }
	  else if (!window.MediaRecorder) {
		  usingOpusMediaRecorderPolyfill = true;
	  }
	  else {
		  usingOpusMediaRecorderPolyfill = window.MediaRecorder === window.OpusMediaRecorder;
	  }
  }
  
  if (usingOpusMediaRecorderPolyfill) {
	  scope.mediaRecorder = new OpusMediaRecorder(mediaStream, options, scope.polyfill);
  }
  else {
	  scope.mediaRecorder = new MediaRecorder(mediaStream, options);
  }
```

Reply:
After thinking about it for a while, I think that removing it is the correct option. Although I will send you another pull request with a proposal based on [the previous comment](https://github.com/ScarletsFiction/SFMediaStream/pull/21#discussion_r697913484)). Feel free to accept or reject it at your discretion 😇 😉

Reply:
Yes, I already thought similar method like your comment 😄, but I haven't seen someone posted an issue about the error when the `MediaRecorder` is given the third parameter. Maybe they mostly using chromium based browser or already using the polyfill with the `encoderWorker.js`.. so they wasn't seen that error 🤔 

I think the current pull request is already OK, Thanks! 😉